<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditMaskWidget</class>
 <widget class="QWidget" name="EditMaskWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>182</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Mask type :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="maskTypeComboBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>No mask</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Segmentation map</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ignore masked areas</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fill masked areas</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Blender</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;no mask loaded&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Refresh mask</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="importMaskButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Use input resolution or larger.</string>
        </property>
        <property name="text">
         <string>Import mask</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Hint: M to show/hide mask</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
